ports/redtrees/Red Trees.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+source $controlfolder/device_info.txt
+
+get_controls
+
+GAMEDIR=/$directory/ports/redtrees
+CONFDIR="$GAMEDIR/conf/"
+
+CUR_TTY=/dev/tty0
+$ESUDO chmod 666 $CUR_TTY
+
+exec > >(tee "$GAMEDIR/log.txt") 2>&1
```

Comment:
Need to replace the logging code with the new standard way.

```suggestion
> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
```